Markdown/developer_product_guide/electronics_and_semiconductors.md

```diff
@@ -17,40 +17,40 @@ The AEDT platform includes ACT Python scripting (including journaling) as well a
   * Twin Builder
 * [PyEDB (PyAnsys)](https://edb.docs.pyansys.com/)
   * EDB
-  * HFSS 3D Layout
+  * HFSS 3D layout
   * Icepak
 
-There is also the Magnet Segmentation Toolkit which operates alongside PyAEDT to provide automation of the segmentation and skew of inner rotor interior permanent magnet and surface permanent magnet motors.
+The Magnet Segmentation Toolkit integrates with PyAEDT to automate the segmentation and skew of inner rotor interior permanent magnet and surface permanent magnet motors.
 
-* [Magnet Segmentation Toolkit (PyAnsys)](https://magnet.segmentation.toolkit.docs.pyansys.com/version/stable/index.html)
+* [Magnet Segmentation Toolkit documentation (PyAnsys)](https://magnet.segmentation.toolkit.docs.pyansys.com/version/stable/index.html)
 
 ### AEDT training material
 
-* [Ansys Electronics Desktop Automation with PyAEDT Getting Started (ALH)](https://www.ansys.com/training-center/course-catalog/electronics/ansys-electronics-desktop-automation-with-pyeadt-getting-started)
+* [Ansys Electronics Desktop Automation with PyAEDT getting started (ALH)](https://www.ansys.com/training-center/course-catalog/electronics/ansys-electronics-desktop-automation-with-pyeadt-getting-started)
 * [Introduction to PyAEDT](https://developer.ansys.com/blog/introduction-pyaedt)
-* [PyAnsys Training: Overview of PyAEDT](https://www.youtube.com/watch?v=yFUboNyJeGk)
-* [PyAnsys Training: PyAEDT Examples and Use Cases](https://www.youtube.com/watch?v=MnACOZ1Axd4)
+* [Overview of PyAEDT: Drive innovation in virtual prototyping with PyAEDT](https://www.youtube.com/watch?v=yFUboNyJeGk)
+* [PyAEDT examples and use cases: Virtual compliance framework](https://www.youtube.com/watch?v=MnACOZ1Axd4)
 
 ### Non-Ansys AEDT training material
 
 * [PyAEDT: Introduction to the Ansys Electronics Desktop API (Ozen)](https://blog.ozeninc.com/resources/pyansys-introduction-to-the-electronics-desktop-api)
 
-## MotorCAD 
+## MotorCAD
 
-* [PyMotorCAD (PyAnsys)](https://motorcad.docs.pyansys.com/)
+* [PyMotorCAD documentation (PyAnsys)](https://motorcad.docs.pyansys.com/)
 
-## ConceptEV 
+## ConceptEV
 
 * [ConceptEV REST API](https://conceptev.ansys.com/api/docs)
-* [PyConceptEV (PyAnsys)](https://conceptev.docs.pyansys.com/)
+* [PyConceptEV documentation (PyAnsys)](https://conceptev.docs.pyansys.com/)
 
-## NuHertz Filter Solutions+
+## Nuhertz Filter Solutions+
 
-See Electronics Desktop.
+See AEDT.
```

Comment:
@jgd10 Saying "See AEDT." doesn't seem sufficient to me as there is no mention of Nunhertz Filter Solutions in this subheading.